updatecli/updatecli.d/terraform.yml

```
@@ -0,0 +1,55 @@
+---
+name: Bump TERRAFORM version
```

```suggestion
name: Bump `terraform` version
```

Nit: indicates a CLI in the markdown based titles